WireUI/Sources/WireMainNavigation/Resources/Localizable.xcstrings

```diff
@@ -374,425 +374,1457 @@
+            "value" : "tabBar.contacts.description"
           }
         },
         "it" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
           }
         },
         "ja" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
           }
         },
         "lt" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
           }
         },
         "nl" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
           }
         },
         "pl" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
           }
         },
         "pt-BR" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
           }
         },
         "ru" : {
           "stringUnit" : {
-            "state" : "translated",
-            "value" : "Список недавних бесед"
+            "state" : "new",
+            "value" : "tabBar.contacts.description"
           }
         },
         "sl" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
           }
         },
         "tr" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
           }
         },
         "uk" : {
           "stringUnit" : {
-            "state" : "translated",
-            "value" : "Список недавніх розмов"
+            "state" : "new",
+            "value" : "tabBar.contacts.description"
           }
         },
         "zh-Hans" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
           }
         },
         "zh-Hant" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "List of recent conversations"
+            "value" : "tabBar.contacts.description"
           }
         }
       }
     },
-    "tabBar.conversations.hint" : {
+    "tabBar.contacts.hint" : {
       "localizations" : {
         "ar" : {
           "stringUnit" : {
             "state" : "new",
-            "value" : "Double tap to open list of recent conversations"
+            "value" : "tabBar.contacts.hint"
```

Comment:
We see the `key` in the value here because the source, `tabBar.contacts.hint`, currently has no values, not even English. - ie.. this is something that needs to be fixed but is not related to the PR.